Sources/Features/Breadcrumb/BacktraceBreadcrumbsLogManager.swift

```diff
@@ -2,39 +2,46 @@ import Foundation
 
 @objc class BacktraceBreadcrumbsLogManager: NSObject {
 
-    private var breadcrumbId = Date().millisecondsSince1970
+    private var breadcrumbId: Int
     private let backtraceBreadcrumbFileHelper: BacktraceBreadcrumbFileHelper
 
-    init(_ breadcrumbLogPath: String, maxQueueFileSizeBytes: Int) throws {
-        self.backtraceBreadcrumbFileHelper = try BacktraceBreadcrumbFileHelper(breadcrumbLogPath,
-                                                                               maxQueueFileSizeBytes: maxQueueFileSizeBytes)
+    init(breadcrumbSettings: BacktraceBreadcrumbSettings) throws {
+        self.backtraceBreadcrumbFileHelper = try BacktraceBreadcrumbFileHelper(breadcrumbSettings)
```

Comment:
What if the backtraceBreadcrumbFileHelper threw an exception? I think we shouldn't continue setting rest information.

Reply:
Try without question mark throws the exception, so it's the constructing class' problem